src/PanelTraits/Create.php

```diff
@@ -120,4 +136,86 @@ public function syncPivot($model, $data, $form = 'create')
             }
         }
     }
+
+    private function createOneToOneRelations($item, $data, $form = 'create')
```

Comment:
Function shouldn't be private. It blocks extensibility as it cannot be overridden. Protected is better.